tests/unit/parallel/algorithms/partition_tests.hpp

```diff
@@ -435,10 +435,12 @@ void test_partition()
         }, rand_base);
 
     ////////// Many test cases for meticulous tests.
+#if !defined(HPX_DEBUG)
```

Comment:
Why is this guarded with `HPX_DEBUG` now?

Reply:
because the heavy tests take an eternity when compiled in debug mode.